src/site/stages/build/page-builder/transformers.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * A very specific helper function that expects to receive an
+ * array with one item which is an object with a single `value` property
+ *
+ */
+function arrayCollapse(arr) {
+  return arr.reduce((value, item) => {
+    // eslint-disable-next-line no-param-reassign
+    value = item.value;
+    return value;
+  }, null);
+}
+
+function pageTransform(entity) {
+  const transformed = entity;
+  const {
+    title,
+    changed,
+    fieldIntroText,
+    fieldPageLastBuilt,
+    fieldAlert,
+    fieldDescription,
+    moderationState: [{ value: published }],
+  } = entity;
+  // collapse title
+  // Question: Can we always assume that title is an array of one item, with that item being an object with a `value` key?
+  transformed.title = arrayCollapse(title);
+  transformed.entityBundle = 'page';
+
+  transformed.fieldIntroText = arrayCollapse(fieldIntroText);
+  transformed.changed = new Date(arrayCollapse(changed)).getTime() / 1000;
+  transformed.fieldPageLastBuilt = new Date(
+    arrayCollapse(fieldPageLastBuilt),
+  ).toUTCString();
+
+  transformed.entityPublished = published === 'published';
+  delete transformed.moderationState;
+
+  if (Array.isArray(fieldDescription) && fieldDescription.length === 0) {
+    transformed.fieldDescription = null;
+  }
+
+  if (
+    Array.isArray(fieldAlert) &&
```

Comment:
[Same here](https://github.com/department-of-veterans-affairs/vets-website/pull/10973/files#r340257303) :)

Reply:
I think this one is a bit different.  `fieldAlert` might be `[[]]` which isn't considered empty.  However, this worked:

```js
if (_.isEmpty(_.flatten(fieldAlert))) {
  // ...
}
```